### PR TITLE
Make validate-rules command a bit more verbose

### PR DESCRIPTION
**Pull request by dims**
Change-Id: I3e9c8eb2542b50f81ba7cc1ea594f2a12384f9dd

### CODE REVIEW

File: cmd/publishing-bot/config/rules.go [hunk 1]
```diff
@@ -118,6 +120,7 @@ func readFromUrl(u *url.URL) ([]byte, error) {
 }
 
 func validateRepoOrder(rules *RepositoryRules) (errs []error) {
+	fmt.Printf("validating repository order\n")
```
Comment (sttts) [inline cmd/publishing-bot/config/rules.go]: we use glog elsewhere if I am not mistaken.

Comment (dims) [reply cmd/publishing-bot/config/rules.go]: ack i can switch it over

Comment (dims) [reply cmd/publishing-bot/config/rules.go]: Done

**Pull request merged**